
Do not source all the .pc and header files - we will pull in a lot of
dependencies and the list would not match the installed one anyway ...

  - Rene Rebe <rene@exactcode.de>

--- anjuta-2.0.0/global-tags/create_global_tags.sh.orig	2005-03-11 18:53:15.000000000 +0100
+++ anjuta-2.0.0/global-tags/create_global_tags.sh	2005-05-20 11:19:23.000000000 +0200
@@ -66,7 +66,7 @@
 
 # pkg-config libraries
 PKG_CONFIG=`which pkg-config 2>/dev/null`
-if ( [ ! -z $PKG_CONFIG ] && [ -x $PKG_CONFIG ] ) ; then
+if false ; then
     
     PKG_DEFAULT_CONFIG_PATH=`echo $PKG_CONFIG | sed -e 's/\/bin\/pkg-config/\/lib\/pkgconfig'/`
     PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$PKG_DEFAULT_CONFIG_PATH"
